models/countries.go

```diff
@@ -0,0 +1,138 @@
+package models
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+const COUNTRIES_API = "https://restcountries.eu/rest/v2/alpha/"
+
+var FILE_PATH = os.Getenv("COUNTRIES_FILE")
+
+type Country struct {
+	Name        string
+	Capital     string
+	Area        float64
+	Gini        float64
+	NativeName  string
+	NumericCode string
+	Population  int
+	Region      string
+	SubRegion   string
+	Code        string
+}
+
+var rows = []string{
+	"Name",
+	"Capital",
+	"Area",
+	"Gini",
+	"NativeName",
+	"NumericCode",
+	"Population",
+	"Region",
+	"SubRegion",
+}
+
+func NewCountry(code string) *Country {
+	return &Country{
+		Code: code,
+	}
+
+}
+
+func (c *Country) Store() error {
+	f, err := os.Create(FILE_PATH + c.Code + ".csv")
```

Comment:
This could be changed later, but move the files to a config file and open them in the main